.github/workflows/ci_build_docker.yml

```diff
@@ -50,3 +55,11 @@ jobs:
         run: mvn --batch-mode --update-snapshots -e -V clean package -DskipTests -Pdocker
         env:
           CI: false
+
+      - name: Push docker image to Docker Hub
+        if: ${{ success() && github.event_name == 'push' && github.repository == 'apache/incubator-inlong' }}
+        run: sh docker/publish.sh
+        env:
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+          DOCKER_REGISTRY: 'hub.docker.com'
```

Comment:
Does the `DOCKER_REGISTRY` is needless, the default registry is `hub.docker.com`.

Reply:
OK, I will remove it.